web/src/app/admin/connectors/[connector]/pages/ConnectorInput/NumberInput.tsx

```diff
@@ -0,0 +1,42 @@
+import { SubLabel } from "@/components/admin/connectors/Field";
+import { Field } from "formik";
+
+export default function NumberInput({
+  label,
+  value,
+  optional,
+  description,
+  name,
+  showNever,
```

Comment:
I'm not entirely clear on why showNever is a part of a Number Input?  Also thinking generally if we want to reuse components, we should TSDoc / JSDoc them.

Reply:
`showNever` is meant to show "never" if 0. Agree that this could be not verbose enough